crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -295,7 +294,7 @@ pub enum AdminRequest {
     /// [`AdminResponse::AgentInfoAdded`]
     AddAgentInfo {
         /// list of signed agent info to add to peer store
-        agent_infos: Vec<AgentInfoSigned>,
+        agent_infos: Vec<String>,
```

Comment:
This will be the canonical json string now. Question: should we make this the default serde encoding/decoding in k2 so the type can be here directly?

Reply:
I'm fine with expecting callers to have this in string format though I'm wondering if `Vec<String>` is right. Doesn't the agent info encode to a JSON array? I don't think it operates on single values because I ran into that. So maybe this should just be a String and then it definitely wants the doc comment updating with some hints about where the string value comes from

Reply:
No, it's a JSON object https://github.com/holochain/kitsune2/blob/main/crates/api/src/agent.rs#L370

We've been using JSON strings as encoding in k2 for agent infos, so it may as well be the default.

Reply:
Yeah, actually we can't implement serde deserialize on it anyways, because you need to be able to specify a verifier... Okay, leaving this as `Vec<String>` for now. We can always revisit.

Reply:
> I don't think it operates on single values because I ran into that.

There is both a [decode](https://docs.rs/kitsune2_api/latest/kitsune2_api/struct.AgentInfoSigned.html#method.decode) and a [decode_list](https://docs.rs/kitsune2_api/latest/kitsune2_api/struct.AgentInfoSigned.html#method.decode_list).